### ISSUE TITLE
Noisy warning while using the DeepHyper CLI

### ISSUE THREAD

**Issue by Deathn0t** [deephyper/deephyper#24]
I quote @felker from #23 
>> Still, curious that DeepHyper loads TensorFlow (and its noisy warnings) even for the --help dialog.

The problem is happening when using:

```bash
$ deephyper --help
```
Indeed these warnings are flooding the shell and are very annoying, I create this issue so that we keep thinking about solving this.

**Comment by felker** [comment]
For my project, I just suppress them in `__init__.py` wiht:

```python
warnings.filterwarnings('ignore',
                        category=FutureWarning,
                        message=r"passing \(type, 1\) or '1type' as a synonym of type is deprecated",
                        module="tensor*")
```

They come from TensorFlow with NumPy >= v1.17.0.

But it is probably better to bypass importing TensorFlow entirely when `--help` is used, if possible.

**Comment by Deathn0t** [comment]
To build the DeepHyper CLI parser we import the search classes and do some introspection on them. So the PPO search using reinforcement learning is imported and it itself imports Tensorflow. I recently noticed that a Mpi4py mock was added to this process to avoid mpi errors  while using the CLI when openmpi is not installed (e.g., Theta login nodes). Thus, the `deephyper nas ppo..` was not working at all.  Given this two issues, I decided to remove the PPO search from the CLI. Tensorflow is not loaded anymore while using the parser.
